koku/masu/database/trino_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -399,7 +398,21 @@ INSERT INTO hive.{{schema | sqlsafe}}.aws_openshift_disk_capacities_temp (
     year,
     month
 )
-WITH cte_ocp_filtered_resources as (
+WITH cte_hours as (
+    SELECT
+        DAY(
+            DATE_ADD(
+                'day',
+                -1,
+                DATE_ADD(
+                    'month',
+                    1,
+                    DATE_TRUNC('month', DATE_PARSE(CONCAT('{{year | sqlsafe}}-', '{{month | sqlsafe}}', '-01'), '%Y-%m-%d'))
+                )
+            )
+        ) * 24 AS in_month
+),
+cte_ocp_filtered_resources as (
```

Comment:
The last_day_of_month function might make some of this logic a little sompler since I think that is what this is gathering. https://trino.io/docs/current/functions/datetime.html#last_day_of_month

Reply:
Yeh thats actually a good idea. I wasnt so hot on this but didnt have a better suggestion. Thanks Corey.